azkaban-common/src/main/java/azkaban/event/EventData.java

```diff
@@ -0,0 +1,34 @@
+package azkaban.event;
+
+import azkaban.executor.Status;
+
+/**
+ * Carries a snapshot of the status data, suitable for asynchronous message passing.
+ */
+public class EventData {
+
+  private Status status;
+  private String nestedId;
```

Comment:
I don't really know, does it require a comment? Previously `Node.getNestedId()` was used in some event handlers, so I'm carrying it here, as the `Node` is replaced with `EventData` in `Event`.